torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -240,6 +241,17 @@ def heavy_rpc_torchscript(tensor):
         tensor /= i + 1
     return 0
 
+
+@torch.jit.script
+def script_sleep_and_add(x):
```

Comment:
why there is a "sleep" in the name?

Reply:
Sorry, changed the name. This was a vestige of my first attempt, where I tried a sleep, until I realize that we can't time.sleep in TorchScript.